### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by MrNavaStar**
Nothing crazy here, just made things feel a bit cleaner. Might try and get the favicon to work at some point as it currently does not.

### CODE REVIEW

**Comment by MrNavaStar** [comment]
Here's what it looks like, lmk what you think
image

File: internal/server/ui/index.html [hunk 1]
```diff
@@ -1,62 +1,84 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
```
Comment (qdm12) [inline internal/server/ui/index.html:2]: nitpicking here; Are these necessary?

```suggestion
<html>

```

File: internal/server/ui/index.html [hunk 2]
```diff
@@ -1,62 +1,84 @@
...
       <th>Host</th>
       <th>Provider</th>
-      <th>IP version</th>
-      <th>Update status</th>
-      <th>Set IP</th>
-      <th>Previous IPs (reverse chronological order)</th>
+      <th>IP Version</th>
+      <th>Update Status</th>
+      <th>Current IP</th>
+      <th>Previous IPs</th>
```
Comment (qdm12) [inline internal/server/ui/index.html:81]: Can we keep the `(reverse chronological order)` somehow? :thinking:

**Review by qdm12 — COMMENTED** [review]
Thank you so much!
Also excuse my late review, I was a bit way-over-busy recently.

Did you check how it looks like on mobile? It would be nice to have something scaling on both desktop and mobile ideally :wink:

File: internal/server/ui/index.html (hunk 1 above)
Comment (MrNavaStar) [reply internal/server/ui/index.html:2]: No doctype sometimes causes errors in the browser console, the lang probably isn't but it is good as we only support 1 localization

File: internal/server/ui/index.html (hunk 2 above)
Comment (MrNavaStar) [reply internal/server/ui/index.html:81]: Yeah we can, it just looked cluttered and seems obvious in my opinion

**Comment by MrNavaStar** [comment]
I didn't look on mobile, kinda forgot about that as I don't use my phone much for those sorts of things. A mobile ui will probably need to be quite different as viewing tables on mobile is always a pain

File: internal/server/ui/index.html (hunk 2 above)
Comment (MrNavaStar) [reply internal/server/ui/index.html:81]: Maybe we could make it show on hover, that would probably be the best of both worlds

**Comment by fuse314** [comment]
Consider not linking the whole font-awesome library just for the Github icon.
Instead, please add the SVG code of the logo directly to the HTML file.

**Pull request closed without merging**